### PR TITLE
Human topic cleanup

### PR DESCRIPTION
**Pull request by ivanmixo**
## About The Pull Request

Whole bunch of 8 year old indent hell and single letter non descriptive vars. Replaced with early returns and better vars.

Yes, this is mostly outdated code for MP and stuff. Suffer.
## Why It's Good For The Game

Cleaner code is better.
## Changelog
:cl:
refactor: Cleaned up the human topic proc
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/human/human.dm [hunk 1]
```diff
@@ -320,354 +320,409 @@
...
+		if(!do_mob(usr, src, HUMAN_STRIP_DELAY, BUSY_ICON_GENERIC, BUSY_ICON_GENERIC))
+			return
+		w_uniform.set_sensors(usr)
 
-			log_combat(usr, src, "attempted to toggle sensors")
-			var/obj/item/clothing/under/U = w_uniform
-			if(U.has_sensor >= 2)
-				to_chat(usr, "The controls are locked.")
+	if(href_list["squadfireteam"])
+		if(usr.incapacitated() || !get_dist(usr, src) <= 7 || !hasHUD(usr,"squadleader"))
```
Comment (QualityVan) [inline code/modules/mob/living/carbon/human/human.dm:437]: dist <= 7 needs parens or the ! will evaluate first
or just flip to > i guess

File: code/modules/mob/living/carbon/human/human.dm [hunk 2]
```diff
@@ -320,354 +320,409 @@
...
+		var/mob/living/carbon/human/H = usr
+		if(!H.mind)
+			return
+		var/obj/item/card/id/ID = get_idcard()
+		if(!ID || !(ID.rank in GLOB.jobs_marines))//still a marine, with an ID.
+			return
+		if(!(assigned_squad == H.assigned_squad)) //still same squad
+			return
+		var/newfireteam = tgui_input_list(usr, "Assign this marine to a fireteam.", "Fire Team Assignment", list("None", "Fire Team 1", "Fire Team 2", "Fire Team 3"))
+		if(!newfireteam || H.incapacitated() || !get_dist(H, src) <= 7) //We might've moved away or gotten incapacitated in the meantime
```
Comment (QualityVan) [inline code/modules/mob/living/carbon/human/human.dm:448]: same here

File: code/modules/mob/living/carbon/human/human.dm [hunk 3]
```diff
@@ -320,354 +320,409 @@
...
-							var/obj/item/dogtag/D = new(loc)
-							D.fallen_names = list(DT.registered_name)
-							D.fallen_assignements = list(DT.assignment)
-							usr.put_in_hands(D)
-						else
-							to_chat(usr, span_warning("You can't take a dogtag's information tag while its owner is alive."))
+		if(usr.incapacitated() || !Adjacent(usr))
+			return
+		if(slot == SLOT_WEAR_ID)
+			if(istype(wear_id, /obj/item/card/id/dogtag))
```
Comment (Lumipharon) [inline code/modules/mob/living/carbon/human/human.dm:332]: Can just make these one line.

File: code/modules/mob/living/carbon/human/human.dm [hunk 4]
```diff
@@ -320,354 +320,409 @@
...
+				var/obj/item/card/id/dogtag/DT = wear_id
+				if(!DT.dogtag_taken)
```
Comment (Lumipharon) [inline code/modules/mob/living/carbon/human/human.dm:334]: early return?

File: code/modules/mob/living/carbon/human/human.dm [hunk 5]
```diff
@@ -320,354 +320,409 @@
...
-					usr.stripPanelUnequip(what,src,slot)
+						to_chat(usr, span_warning("You can't take a dogtag's information tag while its owner is alive."))
 				else
-					what = usr.get_active_held_item()
-					usr.stripPanelEquip(what,src,slot)
+					to_chat(usr, span_warning("Someone's already taken [src]'s information tag."))
+				return
+		//police skill lets you strip multiple items from someone at once.
+		if(!usr.do_actions || usr.skills.getRating(SKILL_POLICE) >= SKILL_POLICE_MP)
+			var/obj/item/what = get_item_by_slot(slot)
```
Comment (Lumipharon) [inline code/modules/mob/living/carbon/human/human.dm:350]: I know this was already here, but lmao var name

**Review by Lumipharon — CHANGES_REQUESTED** [review]
Jesus christ, this file lol.

**Review by Lumipharon — APPROVED** [review]
Probs fine but going to TM in case there's any oversights.

**Pull request merged**